taxi/views.py

```diff
@@ -36,6 +42,21 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(ManufacturerListView, self).get_context_data(**kwargs)
```

Comment:
you don't need to specify class and self(however it was required in 2 python) because Mixin doesn't have this method